bob/src/build_info/build_time.rs

```diff
@@ -0,0 +1 @@
+pub const BUILD_TIME: &str = "time-undefined";
```

Comment:
This file is generated at build time. Perhaps it can be removed from the repo and added to `.gitignore` to prevent committing automatic changes to it